Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -953,7 +953,7 @@ public bool InRangeUnobstructed(
             if (!inRange && popup && _gameTiming.IsFirstTimePredicted)
             {
                 var message = Loc.GetString("interaction-system-user-interaction-cannot-reach");
-                _popupSystem.PopupEntity(message, origin, origin);
+                _popupSystem.PopupClient(message, origin, origin);
```

Comment:
a lot of server side systems call `InRangeUnobstructed` and this will just kill the popup for them because `PopupClient` does nothing on the server
```suggestion
                _popupSystem.PopupEntity(message, origin, origin);
```